lib/commanded/event/handler.ex

```diff
@@ -761,15 +792,22 @@ defmodule Commanded.Event.Handler do
     case opts[:start_from] || :origin do
       start_from when start_from in [:origin, :current] -> start_from
       start_from when is_integer(start_from) -> start_from
-      invalid -> "Invalid `start_from` option: #{inspect(invalid)}"
```

Comment:
Is there a reason this did not have a raise or is it a bug?

Reply:
This is a bug, good catch.